packages/renderer/src/lib/dashboard/ProviderInstalled.svelte

```diff
@@ -134,34 +134,32 @@ function onInstallationClick() {
 
 <ProviderCard provider={provider}>
   <svelte:fragment slot="content">
-    <p class="text-sm text-gray-700 w-2/3 text-center" aria-label="Suggested Actions">
+    <p class="text-sm text-[var(--pd-content-text)] w-2/3 text-center" aria-label="Suggested Actions">
       To start working with containers, {provider.name} needs to be initialized.
     </p>
 
-    <div class="min-w-[230px] w-1/3 flex justify-center" class:hidden={!initializationButtonVisible}>
+    <div class="min-w-[230px] w-1/3 flex justify-center" class:hidden={initializationButtonVisible}>
       <div class="w-[212px] relative">
-        <div class="bg-gray-300 text-white flex w-[212px]">
+        <div class="bg-[var(--pd-invert-content-card-bg)] text-[var(--pd-button-text)] flex w-[212px]">
           <button
-            class="float-left bg-purple-600 hover:bg-purple-500 pt-2 pr-3 pl-3 pb-2 text-[13px] text-white mr-px w-[180px]"
+            class="float-left bg-[var(--pd-button-primary-bg)] hover:bg-[var(--pd-button-primary-hover-bg)] pt-2 pr-3 pl-3 pb-2 text-[13px] text-[var(--pd-button-text)] mr-px w-[180px]"
             on:click={onInstallationClick}>
             {installationOptionSelected}
           </button>
           <button
-            class="inline-block bg-purple-600 hover:bg-purple-500 text-[13px] text-white pt-2 pr-3 pl-3 pb-2 w-[32px]"
+            class="inline-block bg-[var(--pd-button-primary-bg)] hover:bg-[var(--pd-button-primary-hover-bg)] text-[13px] text-[var(--pd-button-text)] pt-2 pr-3 pl-3 pb-2 w-[32px]"
             aria-label="Installation options menu"
             on:click={() => updateOptionsMenu(!installationOptionsMenuVisible)}>
             <i class="fas fa-caret-down"></i>
           </button>
         </div>
         <div
-          class="z-10 min-w-[130px] m-auto bg-primary text-[13px] text-white absolute w-full"
+          class="z-10 min-w-[130px] m-auto bg-primary text-[13px] text-[var(--pd-button-text)] absolute w-full"
           class:hidden={!installationOptionsMenuVisible}>
-          <ul class="w-full outline-none bg-charcoal-800 rounded-sm placeholder-gray-700">
+          <ul class="w-full outline-none bg-[var(--pd-dropdown-bg)] rounded-sm placeholder-[var(--pd-content-text)]">
             <li>
               <button
-                class="w-full p-2 {installationOptionSelected === InitializeOnlyMode
-                  ? 'bg-purple-600 text-white'
-                  : 'bg-purple-700 text-gray-700'} hover:bg-purple-500 cursor-pointer"
+                class="w-full p-2 bg-[var(--pd-button-primary-bg)] text-[var(--pd-button-text)] hover:bg-[var(--pd-button-primary-hover-bg)] cursor-pointer"
```

Comment:
@deboer-tim the tertiary button bg mentioned in https://github.com/containers/podman-desktop/pull/9420#issuecomment-2423121485 was supposed to be used here since I wanted to keep the colors as close as possible to the original hardcoded colors. For this line and L173, this was the hardcoded color design:
![Screenshot from 2024-10-22 15-39-12](https://github.com/user-attachments/assets/80f8ba1f-f60f-41b8-9306-f877da81b982)
so in order to not change the design, I created the tertiary button bg, but TBH, I'm not sure what is the reasoning behind this design, so for now I just used the primary button bg and removed the tertiary color:
![Screenshot from 2024-10-22 15-41-47](https://github.com/user-attachments/assets/5e1d08ac-287f-48b7-8f1a-225825f62650)

If the colors need to be changed for this updated design, lmk and I'll used or add the new colors.

Reply:
TBH, I'm not sure that button ever had a real design, and I agree it's simpler for now to just use the existing colors vs adding a new color that might be unnecessary or asking design to spend time on this. 👍🏼